src/components/images/brandToLogo.tsx

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+
+import masterCardImage from "./masterCard.png";
+import visaImage from "./visa.png";
+import paypalImage from "./PP.png";
+
+export const brandToLogo = (brand: string) => {
+  //h*4+my*8=40
```

Comment:
what does this comment mean?

Reply:
There is a probability that I will need to add another logos here (e.g. american express) and this is the equation that determines the nice ratio of height and margins to make all rows in tokens list look homogeneous.